test/test_auth.py

```diff
@@ -74,8 +77,16 @@ def __init__(self, collection):
         self.success = False
 
     def run(self):
-        assert self.collection.find_one({"$where": delay(1)}) is not None
-        self.success = True
+        if _IS_SYNC:
+            assert self.collection.find_one({"$where": delay(1)}) is not None
+            self.success = True
+        else:
+
+            def _run_async():
```

Comment:
Any threading test needs to be skipped for async because async client doesn't support multithreading.